apps/settings/elements/accessibility.html

```diff
@@ -13,6 +13,12 @@ <h1 data-l10n-id="accessibility-header"></h1>
             <small id="colorFilter-desc">Disabled</small>
           </a>
         </li>
+        <li>
+          <a class="menu-item" href="#accessibility-monoaudio">
+            <span data-l10n-id="monoAudio">Mono Audio</span>
+            <small id="monoAudio-desc">Disabled</small>
```

Comment:
Actually i just noticed that in bug 1202341 we remove strings from HTML completely, so no need for Mono Audio and Disabled in HTML if there are done via data-l10n-id.